lib/course_refresher.rb

```diff
@@ -189,6 +189,8 @@ def check_directory_names
     end
 
     def update_course_options
+      return if @course.refreshed?
```

Comment:
Maybe comment why this 
`#after initial refresh all options are set and updated via web ui`